--- a/tools/winedump/pe.c
+++ b/tools/winedump/pe.c
@@ -2503,9 +2503,9 @@
         printf( "  ExtraRFETableSize                      %#x\n", (int)data->ExtraRFETableSize );
         printf( "  __os_arm64x_dispatch_fptr              %#x\n", (int)data->__os_arm64x_dispatch_fptr );
         printf( "  AuxiliaryIATCopy                       %#x\n", (int)data->AuxiliaryIATCopy );
-        printf( "  __os_arm64x_helper0                    %#x\n", (int)data->__os_arm64x_helper0 );
-        printf( "  __os_arm64x_helper1                    %#x\n", (int)data->__os_arm64x_helper1 );
-        printf( "  __os_arm64x_helper2                    %#x\n", (int)data->__os_arm64x_helper2 );
+        printf( "  AuxiliaryDelayloadIAT                    %#x\n", (int)data->AuxiliaryDelayloadIAT );
+        printf( "  AuxiliaryDelayloadIATCopy                    %#x\n", (int)data->AuxiliaryDelayloadIATCopy );
+        printf( "  HybridImageInfoBitfield                    %#x\n", (int)data->HybridImageInfoBitfield );
         printf( "  __os_arm64x_helper3                    %#x\n", (int)data->__os_arm64x_helper3 );
         printf( "  __os_arm64x_helper4                    %#x\n", (int)data->__os_arm64x_helper4 );
         printf( "  __os_arm64x_helper5                    %#x\n", (int)data->__os_arm64x_helper5 );
